ephemeris/shed_install.py

```diff
@@ -368,6 +288,13 @@ def _parser():
                         help="Galaxy tool panel section label where tool will be installed "
                              "(if the section does not exist, it will be created; "
                              "only applicable if the tools file is not provided).")
+    parser.add_argument("--revisions",
+                        default=None,
+                        dest="revisions",
+                        help="The revisions of the tool repository that will be installed. "
+                             "Revisions must be specified in on the command line in YAML format:"
```

Comment:
s/in//

But this is rather odd for the UX. Can we use `nargs=""` here instead?